helm/hookshot/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Should this be a slightly more readable markdown file with more explanation?

Reply:
This is a default template gen stub - So yeah, it can be expanded upon

Reply:
I don't have any immediate ideas how to improve this TBH - Also, this is what's emitted to the console when the chart is installed / upgraded, not part of the docs itself

Reply:
Eh, I'm happy to leave it as a future thing.